template/assets/use_case/template/README.md.hbs

```diff
@@ -1,3 +1,7 @@
+<!--- score_badge -->
```

Comment:
use case doesn't have a score

Reply:
This is the template returned by the pyproject shouldn't we inject the score here?

Reply:
ahh yup! my bad, didn't read the path correctly!!!